src/pages/uxp/reference-js/Modules/uxp/Plugin Manager/Plugin.md

```diff
@@ -0,0 +1,75 @@
+
+<a name="plugin" id="plugin"></a>
+
+# Plugin
+Plugin object receveied in the PluginManager.getPlugins list,
used for IPC(Inter Plugin Communication)
+
+
+
+<a name="plugin-id" id="plugin-id"></a>
+
+## id ⇒ `String`
+Get plugin id
+
+**Returns**: `String`  
+
+
+<a name="plugin-version" id="plugin-version"></a>
+
+## version ⇒ `String`
+Get plugin version
+
+**Returns**: `String`  
+
+
+<a name="plugin-name" id="plugin-name"></a>
+
+## name ⇒ `String`
+Get plugin name
+
+**Returns**: `String`  
+
+
+<a name="plugin-manifest" id="plugin-manifest"></a>
+
+## manifest ⇒ `JSON`
+Get plugin manifest
+
+**Returns**: `JSON`  
+
+
+<a name="plugin-enabled" id="plugin-enabled"></a>
```

Comment:
Once again: unclear if this is a JSON string or one of the supported JSON data types

Reply:
🔴 🟡

Reply:
Tracked in #81